elide-core/src/main/java/com/yahoo/elide/Elide.java

```diff
@@ -278,7 +283,10 @@ protected ElideResponse handleRequest(boolean isReadOnly, User user,
                                           Supplier<DataStoreTransaction> transaction,
                                           Handler<DataStoreTransaction, User, HandlerResult> handler) {
         boolean isVerbose = false;
+        UUID requestId = null;
```

Comment:
We are storing requestId as a string in requestScope. Can we do the same here? or is there an advantage to storing it like this?
https://github.com/yahoo/elide/pull/1331/files

Reply:
We are using it as a hash value in a Map so I  guess we can store it either way